src/main/java/com/centralesupelec/chowchow/search/web/SearchWebController.java

```diff
@@ -0,0 +1,24 @@
+package com.centralesupelec.chowchow.search.web;
+
+import com.centralesupelec.chowchow.search.controllers.SearchController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(path = "/search")
+public class SearchWebController {
+
+    private final SearchController searchController;
+
+    @Autowired
+    public SearchWebController(SearchController searchController) {
+        this.searchController = searchController;
+    }
+
+    @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I'd have add the path, even if it's `""`, just to keep things clear.

Reply:
I find it clearer to not expose an empty path 🤷‍♂ I guess it depends, let's keep it like that if you don't mind.